pyaedt/icepak.py

```diff
@@ -5741,3 +5750,320 @@ def _assign_blower(self, props, faces, inlet_face, fan_curve_flow_unit, fan_curv
+            The object to which the boundary is assigned. Surface name
+            if str, face id if int.
+        boundary_name : str, optional
+            Boundary name. If not provided, a name is generated
+            automatically. Default is ``None``
+        total_power : str or float or dict, optional
+            Power dissipated by the plate. If float, the default unit is
+            ``"W"``. A transient or temperature dependent power can be
+            assigned with a dictionary. Default is ``"0W"``.
+        thermal_resistance : str or float, optional
+            If ``"Thermal Resistance"`` is specified as the thermal
+            condition, this parameter represents the thermal resistance
+            of the plate. If float, the default unit is ``"Kel_per_W"``.
+            Default is ``"0Kel_per_W"``.
+        solid_material : str, optional
+            If ``"Thickness"`` is specified as the thermal condition,
+            this parameter represents the material of the conducting plate.
+            Default is ``"Al-Extruded"``.
+        shell_conduction : bool, optional
+            Indicates whether shell conduction is considered.
+            Default is ``False``.
+        low_side_rad_material : str, optional
+            Material on the low side for radiation. If ``None``,
+            radiation is disabled on the low side. Default is ``None``.
+        high_side_rad_material : str, optional
+            Material on the high side for radiation. If ``None``,
+            radiation is disabled on the high side. Default is ``None``.
+
+        Returns
+        -------
+        :class:`pyaedt.modules.Boundary.BoundaryObject`
+            Boundary object when successful or ``None`` when failed.
+
+        """
+        return self.assign_conducting_plate(obj_plate,
+                                            boundary_name=boundary_name,
+                                            total_power=total_power,
+                                            thermal_specification="Thickness",
+                                            thickness=thickness,
+                                            solid_material=solid_material,
+                                            shell_conduction=shell_conduction,
+                                            low_side_rad_material=low_side_rad_material,
+                                            high_side_rad_material=high_side_rad_material)
+
+    def assign_conducting_plate_with_resistance(self, obj_plate, boundary_name=None, total_power="0W",
+                                                thermal_resistance="0Kel_per_W",
+                                                shell_conduction=False, low_side_rad_material=None,
+                                                high_side_rad_material=None):
+        """
+        Assign thermal boundary conditions with thickness specification to a conducting plate.
+
+        Parameters
+        ----------
```

Comment:
Please format all subsequent "Parameters" sections with the same descriptions from earlier in the PR. Thanks!